.github/workflows/oe_license.yml

```diff
@@ -0,0 +1,55 @@
+name: OE License
+
+on:
+  push:
+    branches:
+      - "master"
+      - "maintenance/.+"
+  pull_request:
+    branches:
+      - "master"
+      - "maintenance/.+"
```

Comment:
Are these needed?

Reply:
Probably not `maintenance`, but `master` would be needed in case we want to change how these checks are performed (at least `pull_request`).